Storage/storage.h

```diff
@@ -75,26 +75,6 @@ namespace storage {
 // 2) Splits the type into `T_ADDER` and `T_DELETER`, to support seamless persistence of deletions.
 // clang-format off
 
-#ifndef _MSC_VER
```

Comment:
My last change to `struct.h` unified the base-class constructor call under `: SUPER(...)`. Actually, you first made this change for MSVC, so it must work there :)